includes/payment-methods/class-becs-payment-method.php

```diff
@@ -39,6 +39,6 @@ public function __construct( $token_service ) {
 	 * @return string
 	 */
 	public function get_testing_instructions() {
-		return __( '<strong>Test mode:</strong> use the test account number <number>000123456</number><button></button>. Other payment methods may redirect to a Stripe test page to authorize payment. More test card numbers are listed <a>here</a>.', 'woocommerce-payments' );
```

Comment:
I needed to change the markup being generated.
Before: button and test number were two separate elements.
Now: button wraps the test number, to make the click target bigger.